makes.nix

```diff
@@ -13,6 +13,10 @@
       pubKey = "makes.cachix.org-1:HbCQcdlYyT/mYuOx6rlgkNkonTGUjVr3D+YpuGRmO+Y=";
     };
   };
+  calculateScorecard = {
+    enable = true;
+    target = "github.com/fluidattacks/makes";
```

Comment:
is it possible to run on a local target? That way the check can actually review the current change and not the main branch only.

Reply:
By local you mean the author branch? or a local directory?

Reply:
A local directory

Reply:
#794 I will be commenting updates about this on the issue.